Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -171,42 +138,17 @@ public void Roslyn_Settings_TempFolderIsNotSet()
             // Assert
             logger.AssertTargetNotExecuted(TargetConstants.OverrideRoslynAnalysisTarget);
             logger.AssertTargetNotExecuted(TargetConstants.SetRoslynAnalysisPropertiesTarget);
-            AssertExpectedAnalysisProperties(result, "pre-existing.log", "pre-existing.ruleset", string.Empty); // existing properties should not be changed
-            BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, TargetProperties.TreatWarningsAsErrors, "true");
-            BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, TargetProperties.WarningsAsErrors, "CS101");
-        }
-
-        [TestMethod]
-        [Description("Checks the settings are not set if the ruleset does not exist")]
-        public void Roslyn_Settings_RuleSetDoesNotExist()
```

Comment:
No longer relevant as a separate test - the new behaviour is covered by the new "missing settings" test above.